web/client/localConfig.json

```diff
@@ -221,7 +221,14 @@
                         "alwaysVisible": true
                     }
                 }
-            }, "Home", "FeatureEditor", "WFSDownload",
+            }, "Home", {
+              "name":"FeatureEditor",
+              "cfg": {
+                "customEditorsOptions": {
```

Comment:
Why do we need to configure an empty object? Is this necessary to make the plugin work?

Reply:
you are right,, if someone wants an example there is the plugin documented